src/org/traccar/database/NotificationManager.java

```diff
@@ -54,6 +57,15 @@ public void updateEvent(Event event, Position position) {
         }
 
         Set<Long> users = Context.getPermissionsManager().getDeviceUsers(event.getDeviceId());
+        Device device = Context.getIdentityManager().getDeviceById(event.getDeviceId());
+        if (device != null && device.getAttributes().containsKey("notifyOnly")) {
```

Comment:
Can't we use `lookupConfigString` instead? Ideally we want it to be configurable globally in the config instead of per-device.